File: 백준/Silver/1269. 대칭 차집합/대칭 차집합.js
const input = require("fs").readFileSync("/dev/stdin").toString().trim().split("\n");

const StringToNumber = (ele) => {
    return ele.split(" ").map(Number);
}

const [N,M] = StringToNumber(input[0]);
let A = new Set(StringToNumber(input[1]));
let B = new Set(StringToNumber(input[2]));
let answer = new Set([...A, ...B]);

A.forEach(a =>{
    if(B.has(a)) answer.delete(a) 
})

console.log(answer.size)

File: 백준/Silver/1269. 대칭 차집합/대칭 차집합.py
import sys
input = sys.stdin.readline

n_a, n_b = map(int, input().split())
S_a = set(map(int, input().split()))
S_b = set(map(int, input().split()))

print(len(S_a - S_b) + len(S_b - S_a))